lightning/src/util/events.rs

```diff
@@ -172,6 +205,14 @@ pub enum Event {
 		/// transaction.
 		claim_from_onchain_tx: bool,
 	},
+	/// Used to indicate that a channel with the given `channel_id` is in the process of closure.
+	ChannelClosed  {
+		/// The channel_id which has been barren from further off-chain updates but
```

Comment:
This is a bit strange wording, maybe simply "The channel_id of the channel which has been closed. Note that on-chain transactions resolving the channel are likely still awaiting confirmation."